febus982/rpi-kernel
# add support for building multiple kernel flavours

This PR adds support for building multiple kernel flavours maintaining the existing functionalities of the script.
The change features a list of supported kernel flavours, if an unsupported version is specified the command will exit and give feedback to the user.
By default the script builds all supported kernel flavours.

Tested:

1. ✅ Building kernel flavour `v8`
2. ✅ Building kernel flavour `2712`
3. ✅ Building selected kernel flavours `v8` and `2712` in the same execution
4. ✅ Building all supported kernel flavours `v8` and `2712` in the same execution
5. ✅ Rebuilding kernel flavours automatically skips the ones already done 

Not tested yet:

1. 🔄 Checking is apache package serving still works as expected

Possible improvements:

1. Add guidelines on how to add support for more kernel flavours (ie. generating a config-overlay file)
2. Add guidelines on how to make use of the apt repository
